master/getting-started/kubernetes/installation/hosted/kubeadm/index.md

```diff
@@ -2,41 +2,68 @@
 title: Kubeadm Hosted Install
 ---
 
-This document outlines how to install Calico, as well as a as single node
-etcd cluster for use by Calico on a Kubernetes cluster created by kubeadm.
+This document outlines how to install Calico on a kubeadm cluster.
 If you have already built your cluster with kubeadm, please review the
 [Requirements / Limitations](#requirements--limitations) at the bottom of
 this page. It is likely you will need to recreate your cluster with the
 `--pod-network-cidr` and `--service-cidr` arguments to kubeadm.
 
-Users who have deployed their own etcd cluster outside of kubeadm should
-use the [Calico only manifest](../hosted) instead, as it does not deploy its
-own etcd.
-
 You can easily create a cluster compatible with this manifest by following [the official kubeadm guide](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
 
+## Installation
 
-#### Installation
+The below manifests are compatible with kubeadm clusters initialized with a
+`pod-network-cidr` matching the default pool of `192.168.0.0/16`, as follows:
+
+```shell
+kubeadm init --pod-network-cidr=192.168.0.0/16
+```
+
+### ETCD Datastore
```

Comment:
I am not sure about the all caps for etcd....I believe that it is always lowercased `etcd`. I am seeing it that way even when it begins a sentence, so we should stick to that convention as well. https://github.com/coreos/etcd Also, I am going to be recommending sentence case for headings, (to be discussed soon), so I would recommend a lowercase for datastore as well.

Reply:
Yeah, it should be `etcd`